specs/apm-server.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+inputs:
+  - name: apm
+    description: "APM Server"
+    platforms:
+      - linux/386
+      - linux/amd64
+      - linux/arm64
+      - linux/ppc64le
+      - darwin/amd64
+      - darwin/arm64
+      - windows/amd64
+      - windows/arm64
+      - container/386
+      - container/amd64
+      - container/arm64
+    outputs:
+      - elasticsearch
+      - kafka
+      - logstash
+      - redis
+    command:
+      args:
+        - "-E"
+        - "management.enabled=true"
+        - "-E"
+        - "gc_percent=${APMSERVER_GOGC:100}"
```

Comment:
@blakerouse I am curious here, how this work? When the replacement occurs in the current code?

Reply:
No replacement works in our code. That comes from libbeat being able to replace it from go-ucfg. So Elastic Agent just passes this straight through. If they have `APMSERVER_GOGC` set in there environment say through systemd, then it will get that value.

I was just copying this over from the v1 specification and make it consistent for all beats. I would prefer to remove it, but I did not want to break something by removing it either.

Reply:
Oh yeah good old ucfg environment replacement.  Let's keep it that way.